tools/crs_converter/CRSconverter.xml

```diff
@@ -0,0 +1,316 @@
+<tool id='crs_converter' name='CRSconverter' version='1.0.0+galaxy@VERSION_SUFFIX@' profile='21.05'>
+    <description>to modify Coordinate Reference Systems from a shapefile</description>
+    <macros>
+        <token name='@VERSION_SUFFIX@'>0</token>
+        <import>CRS_macros.xml</import>
+    </macros>
+    <requirements> 
+        <requirement type='package' version='4.3.3'>r-base</requirement> 
+        <requirement type='package' version='1.0-16'>r-sf</requirement>
+        <requirement type='package' version='1.3.1'>r-tidyr</requirement>
+    </requirements>
+    <command detect_errors='exit_code'><![CDATA[
+        Rscript
+            '$__tool_directory__/CRSconverter.R'
+            $os.path.join($input_file.extra_files_path, 'shapefile.shp')
+            '$output_file_format'
+            #for $param_name in $input_param.keys() ## this loop successively calls all the names and values of the PROJ string for each option of the conditional, it avoids a 100 lines command.
+                #if $param_name!='__current_case__'
+                    #if $param_name=='south' and $input_param[$param_name]=='false'
+                        #pass
+                    #elif $param_name=='south' and $input_param[$param_name]=='true'
+                        ' ' '+' $param_name
```

Comment:
it would be : ' +' $param_name parameters and '+' are seperatated with others elements but not between themselves,  blank spaces and '+' are separated because the R script needs to ignore the 1st blank space given as argument to create a proper PROJ string. this would work: 

```
#if $param_name!='__current_case__'   
    #if $param_name=='south' and $input_param[$param_name]=='false'  
        #pass  
    #elif $param_name=='south' and $input_param[$param_name]=='true'
        ' +' $param_name
    #elif $param_name=='approx' and $input_param[$param_name]=='false'
        #pass
    #elif $param_name=='approx' and $input_param[$param_name]=='true'
        ' +' $param_name
    #elif $param_name=='lat_ts' and $input_param[$param_name]=='0'
        #pass
    #else
        ' ' '+' $param_name '='
        $input_param[$param_name]
#end if 
```